packages/calcite-design-tokens/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@esri/calcite-design-tokens",
+  "version": "1.0.0",
+  "description": "Esri's Calcite Design System Tokens",
+  "homepage": "https://github.com/Esri/calcite-design-system",
+  "keywords": [
+    "Calcite",
+    "Calcite Design System",
+    "design tokens",
+    "tokens"
+  ],
+  "main": "./dist/js/calcite-headless.js",
+  "files": [
+    "./dist",
+    "./src",
+    "README.md"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Esri/calcite-design-system.git"
+  },
+  "author": {
+    "name": "Esri"
+  },
+  "license": "SEE LICENSE IN README.md",
+  "bugs": {
+    "url": "https://github.com/Esri/calcite-design-system/issues"
+  },
+  "scripts": {
+    "build:tokens": "node support/run.js",
+    "build:ts": "tsc --project tsconfig.prod.json",
+    "build": "npm run build:ts && npm run build:tokens",
+    "clean": "rimraf dist",
+    "lint:eslint": "eslint --ext .ts,.tsx --fix .",
+    "lint:markdown": "markdownlint --fix --disable MD024 MD013 MD041 MD033",
```

Comment:
This isn't working because it needs to specify a file/dir/glob. Can you change it to what we have in CC, at least until we get the shared linting configs set up.

```json
"lint:markdown": "markdownlint \"**/*.md\" --fix --ignore node_modules --disable MD024 MD013 MD041 MD033",
```

Reply:
Also should the `clean` script remove `support/dist` too?

Reply:
I still think having a dist generated within `support/` is strange. Plus it's hidden, which results in stuff like forgetting to add it to the `clean` script. maybe we could generate it as something like `dist_transformer/` in the root of the package instead?

Reply:
yeah the whole support build was wonky and based off an idea we don't need yet. Cleaned the whole thing up to work with ts-node instead and simplified the structure.